use-cases/manual-org-three-way.md

```diff
@@ -339,13 +436,3 @@ ExecutionRole
 ]
 }
 -->
-
-
-## Testing
-
-Check within Sysdig Secure
-- Integrations > Cloud Accounts
-- Insights > Cloud Activity
-
-- [Official Docs Check Guide](https://docs.sysdig.com/en/docs/installation/sysdig-secure-for-cloud/deploy-sysdig-secure-for-cloud-on-gcp/#confirm-the-services-are-working)
-- [Forcing events](https://github.com/sysdiglabs/terraform-google-secure-for-cloud#forcing-events)
```

Comment:
can we add a small section with pointers to doc on howto test?
we can put it on the `../README.md` file, and optionally leave a link on each use-case